src/main/java/com/scalar/db/storage/rpc/GrpcStorage.java

```diff
@@ -0,0 +1,207 @@
+package com.scalar.db.storage.rpc;
```

Comment:
This is more like a client SDK?
I'm wondering if it shouldn't be under `storage`.

Reply:
As discussed, we will keep it for now.